### ISSUE TITLE
Unloader, Plastanium 

### ISSUE THREAD

**Issue by JanStruan** [Anuken/Mindustry#4466]
**Platform**: *Android/iOS/Mac/Windows/Linux*
Windows 10 x64 Version 2004
**Build**: *The build number under the title in the main menu. Required. "LATEST" IS NOT A VERSION, I NEED THE EXACT BUILD NUMBER OF YOUR GAME.*
Build 123.1

**Issue**: *Explain your issue in detail.*
First saw in .IO multiplayer survival server. Then recreated in a Custom Game. 
Was making cryofluid using unloader plast conveyor combo. Used Overflow gate to split plastinium conveyor lanes. Split plast lanes did NOT unload into cryofluid makers if they were right NEXT to the overflow gate.  

**Steps to reproduce**: *How you happened across the issue, and what exactly you did to make the bug happen.*
See above in issue

**Link(s) to mod(s) used**: *The mod repositories or zip files that are related to the issue, if applicable.*
NONE, Vanilla Mindustry

**Save file**: *The (zipped) save file you were playing on when the bug happened. THIS IS REQUIRED FOR ANY ISSUE HAPPENING IN-GAME OR IN MULTIPLAYER, REGARDLESS OF WHETHER YOU THINK IT HAPPENS EVERYWHERE. DO NOT DELETE OR OMIT THIS LINE UNLESS YOU ARE SURE THAT THE ISSUE DOES NOT HAPPEN IN-GAME.*
*******(First time using Github/reporting a bug, zipfile is stated to be in crash log area by client)*******
If you remove the line above without reading it properly and understanding what it means, I will reap your soul. Even if you're playing on someone's server, you can still save the game to a slot.

**(Crash) logs**: *Either crash reports from the crash folder, or the file you get when you go into Settings -> Game Data -> Export Crash logs. REQUIRED if you are reporting a crash.*
NO crash [logs]
[BUG REPORT.zip](https://github.com/Anuken/Mindustry/files/5894747/BUG.REPORT.zip)

EDITED: I believe it is a bug, because from what I have seen Anuke has directed Mindustry, this does not make sense.
TLDR; I aint no snek engineer but shit dont make no damn sense
---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  
- [ X] **I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.**
- [ X] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

**Comment by Anuken** [comment]
Loading docks cannot be unloaded.
